app/api/datastore/entities/v1/maat/application.rb

```diff
@@ -3,7 +3,10 @@ module Entities
     module V1
       module MAAT
         class Application < BaseApplicationEntity
+          unexpose :interests_of_justice
+
           expose :submitted_at, as: :declaration_signed_at
+          expose :ioj_bypass, proc: ->(_) { interests_of_justice.empty? }
```

Comment:
I think there is an `if:` argument that you could possibly use rather than a proc - but totally fine either way 👍 

https://github.com/ruby-grape/grape-entity#conditional-exposure